Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -110,6 +110,13 @@ static NugetHelper()
             insideInitializeOnLoad = true;
             try
             {
+                if (SessionState.GetBool("FirstProjectOpen", false))
```

Comment:
Since the `SessionState` is global can we modify the key to help avoid any clashes? Maybe `NugetForUnity.FirstProjectOpen`?

Reply:
Agreed, and done!